examples/netstate-client/main.go

```diff
@@ -54,6 +54,7 @@ func main() {
 			},
 			InlineSegment: []byte("granola"),
 		},
+		XApiKey: []byte("abc13"),
```

Comment:
I think this is meant to be "abc123"?

Reply:
ill leave the 13 here; it's just for testing to show authorization works